lighthouse-core/config/config.js

```diff
@@ -246,8 +246,24 @@ function expandArtifacts(artifacts) {
   return artifacts;
 }
 
+/**
+ * @param {LH.Flags=} flags
```

Comment:
maybe add a function description that this is dropping any flag not found on settings?